dev/io.openliberty.org.jboss.resteasy.server/src/io/openliberty/org/jboss/resteasy/server/component/RESTfulServletContainerInitializer.java

```diff
@@ -45,8 +45,8 @@
     private final static String IBM_REST_SERVLET_NAME = "com.ibm.websphere.jaxrs.server.IBMRestServlet";
     private final static String RESTEASY_DISPATCHER_NAME = "org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher";
     private final static String RESTEASY_DISPATCHER_30_NAME = "org.jboss.resteasy.plugins.server.servlet.HttpServlet30Dispatcher";
-    private final static String APPLICATION = "javax.ws.rs.Application";
-    private final static String EE8_APP_CLASS_NAME = transformBack("javax.ws.rs.core.Application");
```

Comment:
I don't think this line should have changed, but if you aren't going to call transformer any longer, you can just leave this say javax and don't need the transformBack method.